pkg/namespacestore/reconciler.go

```diff
@@ -771,5 +833,55 @@ func (r *Reconciler) ReconcileNamespaceStore() error {
 			return err
 		}
 	}
+
+	if r.CreateNamespaceResourceParams.NSFSConfig != nil {
+		pvcName := r.NamespaceStore.Spec.NSFS.PvcName
+		if !util.KubeCheck(r.DeploymentEndpoint) {
+			r.Logger.Infof("No Endpoint deployment yet. requeuing for 5 minutes")
+			return fmt.Errorf("No Endpoint Deployment. requeue again")
+		}
+		isPvcExist := false
```

Comment:
might be Esyer to call it `willAddPvc` and switch the logic 
WDYT?

Reply:
changed all logic - not relevant